package.json

```diff
@@ -2,10 +2,13 @@
   "name": "pomodoro-react",
   "version": "0.1.0",
   "private": true,
-  "homepage": "https://luizbatanero.github.io/pomodoro-react",
+  "homepage": "https://tiagocunhaf.github.io/pomodoro-react",
```

Comment:
Can you change this url back?

Reply:
Sorry, I change this settings just for "my version"....